apps/api/v1/pages/api/event-types/[id]/_delete.ts

```diff
@@ -51,7 +51,7 @@ async function checkPermissions(req: NextApiRequest) {
   if (isSystemWideAdmin) return;
   /** Only event type owners can delete it */
   const eventType = await prisma.eventType.findFirst({ where: { id, userId } });
-  if (!eventType) throw new HttpError({ statusCode: 403, message: "Forbidden" });
+  if (!eventType) throw new HttpError({ statusCode: 404, message: `Event type with ID ${id} not found` });
```

Comment:
For team events we are no longer connecting userId to event type #16058 . 

In case eventType is a team event, you should check if current user has necessary permissions. You can refer other endpoints to see how we are handling this.